ios/FluentUI.Demo/FluentUI.Demo/Demos/NavigationControllerDemoController.swift

```diff
@@ -54,6 +60,25 @@ class NavigationControllerDemoController: DemoController {
         container.addArrangedSubview(createButton(title: "Change the style every second", action: #selector(showSearchChangingStyleEverySecond)))
     }
 
+    let gradient: CAGradientLayer = {
+        let redColor = CGColor(red: 0.68, green: 0.49, blue: 0.88, alpha: 0.4)
```

Comment:
[nit: This looks more purple than red.](https://www.wolframalpha.com/input?i=RGBColor%5B0.68%2C+0.49%2C+0.88%5D)